huaweicloud/services/acceptance/workspace/resource_huaweicloud_workspace_desktop_test.go

```diff
@@ -228,6 +285,58 @@ resource "huaweicloud_workspace_desktop" "test" {
 `, testAccDesktop_base(rName), rName)
 }
 
+func testAccDesktop_withEPSId(rName, epsId string) string {
+	return fmt.Sprintf(`
+%[1]s
+
+locals {
+  data_volume_sizes = [50, 90, 20, 40]
```

Comment:
I think we can not care the data volumes in this acceptance test, please check it.

Reply:
down